src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -31,29 +31,32 @@ public LineResponse saveLine(LineRequest request) {
 		Station upStation = stationService.findById(request.getUpStationId());
 		Station downStation = stationService.findById(request.getDownStationId());
 		Line persistLine = lineRepository.save(
-			new Line(request.getName(), request.getColor(), upStation, downStation,
+			Line.of(request.getName(), request.getColor(), upStation, downStation,
```

Comment:
어떻게 사용하는게 더 좋은 건 아니지만
저는 이렇게 사용하기도 합니다 :)

```suggestion
			lineRepository.save(request.toLine());
```

Request(dto) 객체에서는 엔티티를 알아도된다고 생각해서 간결하게 쓰기위해서 이렇게 하는편이에요 :)
엔티티가 변경되면 어짜피 바뀌어야 될 객체라고 생각해서(강한결합) 엔티티를 알아도 괜찮을거라 판단했습니다~

---

아래 코드를 보다보니 이미 이런식으로 되어있는것도 있네요 :)
그렇다면 공통적으로 주현님의 컨벤션을 맞춰주시는것도 좋겠습니다.